cdap-common/src/main/java/co/cask/cdap/common/kerberos/ImpersonationRequest.java

```diff
@@ -27,10 +28,19 @@
 public class ImpersonationRequest {
   private final NamespacedEntityId entityId;
   private final ImpersonatedOpType impersonatedOpType;
+  private final String principal;
+  private final String keytabURI;
 
-  public ImpersonationRequest(NamespacedEntityId entityId, ImpersonatedOpType impersonatedOpType) {
+  public ImpersonationRequest(NamespacedEntityId entityId, ImpersonatedOpType impersonatedOpType,
+                              @Nullable String principal, @Nullable String keytabURI) {
```

Comment:
I thought we were not going to send the keytabURI in the request from container to master?